### DISCUSSION TITLE
Docker Desktop replacement

### DISCUSSION THREAD

**Discussion by LadislavPataki** [testcontainers/testcontainers-dotnet#578] [Q&A] [answered]
How to configure Testcontainers to use something else than Docker Desktop on Windows using .Net. Everything is working properly when I'm using Docker Desktop but I'm looking for alternatives. I've already installed Minikube and wanted to update the _.testcontainers.properties_ configuration file, but I wasn't able to find it (TestContainers are added to the project as nuget package). Could you help me with that?

**Comment by HofmeisterAn** [comment, marked as answer]
I do not know much about the configuration without Docker Desktop. I have not tried any other engine expect Rancher Desktop for a quick test. Did you set either the `DOCKER_HOST` environment variable or the Java property `docker.host` the in `~/testcontainers.properties` file?

> I've already installed Minikube and wanted to update the .testcontainers.properties configuration file, but I wasn't able to find it

You have to create the file yourself.

Does the Minikube setup rely on TLS authentication? Then you need to wait until we are done with #370.
